recipes/vectorclass/all/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
```

`FATAL_ERROR` is ignored by CMake >=2.6.
```suggestion
cmake_minimum_required(VERSION 3.0)
```